microk8s-resources/actions/disable.kubeflow.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -eu
+
+source $SNAP/actions/common/utils.sh
+
+
+CONTROLLER="uk8s"
+MODEL="kubeflow"
+
+echo "Removing Kubeflow"
+"$SNAP/microk8s-juju.wrapper" destroy-controller $CONTROLLER --destroy-all-models --destroy-storage || true
```

Comment:
I thought we need a -y here also?

Reply:
I left it off in the script that @ktsakalozos is adapting here, since the user will then get prompted about destroying things